_example/ivfflat/ivfflat.go

```diff
@@ -37,13 +37,7 @@ func main() {
 		xq[i*d] += float32(i) / 1000
 	}
 
-	indexF, err := faiss.IndexFactory(d, "IDMap2,IVF100,SQ8", faiss.MetricInnerProduct)
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer indexF.Close()
```

Comment:
The intention behind updating this example was to bolster confidence in using the new SearchWithoutIDs() API with the same index type FTS uses, with similar steps.
Since we don't use this index type and related APIs(GetSubIndex()), modified it this way.

Reply:
Pls resolve if this makes sense.

Reply:
You misunderstand - I was referring to to `defer index.Close()`